apps/sms/views/shared/test/unit/utils_test.js

```diff
@@ -1153,6 +1153,157 @@ suite('Utils', function() {
     });
   });
 
+  suite('Utils.throttle', function() {
+    setup(function() {
+      this.sinon.useFakeTimers(1);
+    });
+
+    test(
```

Comment:
nit: For the unit test here, we don't split the description and function into 2 different unless the description is too long. So you can simply `test('description', function() {` Sorry for about  confusing :/